docs/server/routes/components/threads/demoComponents/ThreadsWithResovledTab.tsx

```diff
@@ -28,6 +28,7 @@ function ThreadsWithResolvedTab({ groupID }: ThreadsProps) {
     setCurrentTab(tab as TabType);
   }, []);
 
+  // @ts-expect-error
```

Comment:
Not sure what's happening here. Had to ignore this error, otherwise the SDK wouldn't build.